test/e2e/lib/gutenberg/blocks/youtube-block-component.js

```diff
@@ -11,6 +11,23 @@ import GutenbergBlockComponent from './gutenberg-block-component';
 class YoutubeBlockComponent extends GutenbergBlockComponent {
 	static blockTitle = 'YouTube';
 	static blockName = 'core/embed';
+	static blockFrontendSelector = By.css(
+		'.entry-content figure.wp-block-embed iframe.youtube-player'
+	);
+
+	async embed( videoURL ) {
+		const blockDivSelector = `div[id='${ this.blockID.slice( 1 ) }']`;
```

Comment:
`div[id='${ this.blockID.slice( 1 ) }']` can be just `this.blockID` (see [this comment](https://github.com/Automattic/wp-calypso/pull/45796#discussion_r500856393)).